mllib/src/main/scala/org/apache/spark/ml/classification/MultilayerPerceptronClassifier.scala

```diff
@@ -273,29 +273,29 @@ object MultilayerPerceptronClassifier
  * Each layer has sigmoid activation function, output layer has softmax.
  *
  * @param uid uid
- * @param layers array of layer sizes including input and output layers
+ * @param modelLayers array of layer sizes including input and output layers
  * @param weights the weights of layers
  */
 @Since("1.5.0")
 class MultilayerPerceptronClassificationModel private[ml] (
     @Since("1.5.0") override val uid: String,
-    @Since("1.5.0") val layers: Array[Int],
+    @Since("1.5.0") val modelLayers: Array[Int],
```

Comment:
same question, why need to change this?

Reply:
oh, i see. MultilayerPerceptronParams has layers too?

Reply:
Can we rename layers of MultilayerPerceptronParams instead?

Reply:
but renaming layers of MultilayerPerceptronParams will also break the API.